docs/src/content/docs/ja/showcase.mdx

```diff
@@ -17,11 +17,35 @@ Starlightはすでに本番環境で使用されています。以下はウェ
 
 [Starlightを使用しているGitHub上のすべてのパブリックプロジェクトのリポジトリ](https://github.com/withastro/starlight/network/dependents)も確認してください。
 
-## コミュニティプラグイン
+## プラグイン
+
+[プラグイン](/ja/reference/plugins/)により、Starlightの設定、UI、動作をカスタマイズできます。共有や再利用をおこなうことも簡単です。Starlightチームによってサポートされている公式プラグインと、Starlightユーザーによってメンテナンスされているコミュニティプラグインでサイトを拡張しましょう。
+
+### 公式プラグイン
+
+<CardGrid>
+	<LinkCard
+		href="/guides/site-search/#algolia-docsearch"
```

Comment:
```suggestion
		href="/ja/guides/site-search/#algolia-docsearch"
```

The link to this official plugin is actually in the docs so it can be localized.